compilation/src/main/scala/scala/tools/nsc/ScalacBenchmark.scala

```diff
@@ -148,7 +126,7 @@ object ScalacBenchmarkStandalone {
 @BenchmarkMode(Array(SingleShotTime))
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 // TODO -Xbatch reduces fork-to-fork variance, but incurs 5s -> 30s slowdown
-@Fork(value = 16, jvmArgs = Array("-XX:CICompilerCount=2", "-Xms2G", "-Xmx2G"))
+//@Fork(value = 16, jvmArgs = Array("-XX:CICompilerCount=2", "-Xms2G", "-Xmx2G"))
```

Comment:
we cannot "just" change this now, because the results after that change would probably be no longer comparable with the measurements we made so far.